ui/src/app/machines/components/TagLinks/TagLinks.tsx

```diff
@@ -2,13 +2,16 @@ import { Link } from "react-router-dom";
 
 import { filtersToQueryString } from "app/machines/search";
 
-type Props = { tags: string[] };
+type Props = {
+  filterType: "storage_tags" | "tags";
```

Comment:
Do you think it might be helpful to add a comment here for what these different filters are used for?

Reply:
Yep! Done